alerts/openshift-container-storage-operator/helpers/gather_logs.md

```diff
@@ -0,0 +1,5 @@
+(Optional log gathering)
```

Comment:
file already present in helpers folder

Reply:
It seems you need to add the link to "helpers/gatherLogs.md"

Reply:
Corrected